### ISSUE TITLE
Symfony 4 support

### ISSUE THREAD

**Issue by marijnotte** [vihuvac/recaptcha-bundle#15]
Symfony 4 support would be great :)

**Comment by vihuvac** [comment]
@marijnotte Yeah! I've been considering a new versioning in order to support Symfony 4 but I've been a bit busy lately and I had not much time to perform some updates, I just pushed some minor updates and feautres.

I think the bundle may work with Symfony 4 setting some custom params within the settings under ```vihuvac_recaptcha```.

I hope to have some updates soon!

Of course, everyone is encouraged to perfom a PR in order to contribute to the project.

**Comment by clash82** [comment]
yeah, I'd love to see it too.
